src/themes/amsterdam/global_styling/mixins/shadow.ts

```diff
@@ -65,7 +65,12 @@ export const euiShadowMedium = (
   ];
 
   if (property === 'filter') {
-    return `filter: ${array.reduce((v, i) => `${v} drop-shadow(${i})`, '')};`;
+    // To prevent Safari from rendering multiple weird shadows we can only use one drop-shadow
```

Comment:
Minor copy suggestion based on our above discussion
```suggestion
    // Using only one drop-shadow filter instead of multiple is more performant & prevents Safari bugs
```